sycl/test-e2e/AOT/cpu.cpp

```diff
@@ -7,7 +7,6 @@
 //===---------------------------------------------------------------------===//
 
 // REQUIRES: opencl-aot, cpu
-// REQUIRES: build-and-run-mode
```

Comment:
i think i mentioned this in another pr but i think we should define what our goal is based on real world usage. we should decide what cases we need to support, and what cases we think are rare/complicated enough that we can just consider them unsupported for this feature

also i wonder if we can do anything smart to minimize the vars users need to set, like if we see `UNSUPPORTED: hip_amd` maybe we can automatically not compile for `amdgcn-amd-amdhsa` at the build step